hyperspy/drawing/signal.py

```diff
@@ -23,6 +23,8 @@
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 from traits.api import Undefined
 
+from hyperspy.drawing.utils import set_axes_decor
+
 
 def _plot_quiver_scatter_overlay(image, axes_manager,
```

Comment:
Yes, it would be good to remove this function and possibly also `_plot_factors_or_pchars`, which is in the same case.
Is there a way to make this find-able easily from the history if somebody want to use it later? Maybe making an issue that link to it?